habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -1110,6 +1110,24 @@ def _eval_checkpoint(
 
                         rgb_frames[i] = []
 
+                    # A gfx-replay list of keyframes for the episode. This is a JSON string that
```

Comment:
Put this code in a separate helper method and put these comments as method comments. This method is too long already.
Also, I realize this code is only called during evaluation, I think the associated measures and config keys should have the prefix EVAL to make it clear the gfx will not be used during training.